doc/source/user_guide/enhancingperf.rst

```diff
@@ -247,7 +247,7 @@ We've gotten another big improvement. Let's check again where the time is spent:
 
 .. ipython:: python
 
-   %%prun -l 4 apply_integrate_f(df["a"].to_numpy(), df["b"].to_numpy(), df["N"].to_numpy())
```

Comment:
`UsageError: %%prun is a cell magic, but the cell body is empty. Did you mean the line magic %prun (single %)?`

Reply:
Yes, this is a good catch, thanks